ts/Series/Lollipop/LollipopSeries.ts

```diff
@@ -119,16 +147,32 @@ class LollipopSeries extends DumbbellSeries {
     public options: LollipopSeriesOptions = void 0 as any;
     public points: Array<LollipopPoint> = void 0 as any;
 
-    /* *
+    /**
+     * Extend the arearange series' drawPoints method by applying a connector
+     * and coloring markers.
+     * @private
      *
-     *  Functions
+     * @function Highcharts.Series#drawPoints
      *
-     * */
+     * @param {Highcharts.Series} this The series of points.
+     *
+     */
+    public drawPoints(): void {
+        const series = this,
+            pointLength = series.points.length;
 
-    public toYData(point: LollipopPoint): Array<number> {
-        return [pick(point.y, point.low)];
-    }
+        let i = 0,
+            point;
+
+        super.drawPoints.apply(series, arguments as any);
 
+        // Draw connectors
+        while (i < pointLength) {
+            point = series.points[i];
+            series.drawConnector(point as any);
```

Comment:
Can we fix the typing of `drawConnector` so we don't need to cast to `any`?

Reply:
To remove `any` I had to modify `drawConnector` and `getConnectorAttribs` methods from `DumbbellSeries.ts` file. Now everything is working correctly, is that a good approach to add a `LollipopPoint` type to the `Dumbbell` series file?

Reply:
I think that is okay - as long as we only import types, the file size is not affected. What is the best organization is another matter. Ideally they should probably both import some ConnectorSeries types or something. But your suggestion is probably fine.